process-automation/latest/using/index.md

```diff
@@ -174,3 +174,87 @@ Once you have customized a filter, there are two options:
 * **Save filter as**: Selecting this will give you the option to provide a name for a new view for your filter and add it under the **Tasks** section.
 
 You can use the **Delete filter** option at any time to remove a view.
+
+## Condition builder
+
+The condition builder is a tool that helps you build a JUEL expression for a condition. The condition is composed of a set of boolean statement(s) that are linked by an operator. This means the condition to be created is a statement evaluated as a boolean value. The following is an example.
+
+![Condition builder]({% link process-automation/images/condition-builder.png %})
+
+* Click the **$** button to switch the condition builder to an expression editor where you can display and edit the JUEL expression.
+
+* Click the **+** button to add a new statement editor.
+
+* The available operators are:
+  * Every: All statements must be true for the condition to be evaluated as true.
+  * At least one: The condition will be evaluated to be true when one or more of the statements are true.
+  * None: If all the statements are false, then the condition will be evaluated to be true.
+  **Note:** The operator between statements is the same for all the statements.
```

Comment:
This note should be under the operators' bullets so you have to insert a new line and delete the spaces before
```suggestion

**Note:** The operator between statements is the same for all the statements.
```